lib/init.ts

```diff
@@ -1,6 +1,13 @@
+import { isUndefined, isString } from "./utils/index";
+
+type InsightRegion = "de" | "us";
+const SUPPORTED_REGIONS = ["de", "us"];
```

Comment:
The type is not correct, the compiler won't complain for `['de', 'us', 'whatever']`. You can enforce it with an array of `InsightRegion`.

Reply:
I don't understand what you mean.
on the playground when I write [this](https://www.typescriptlang.org/play/index.html#src=type%20A%20%3D%20'us'%20%7C%20'de'%3B%0D%0Aconst%20a%3A%20A%20%3D%20'emea'%0D%0A) it complains

Reply:
You have to add `const SUPPORTED_REGIONS: InsightRegion[]  = ["de", "us"];`